test/tests_for_sdkmr/EE2Status_test.py

```diff
@@ -128,3 +131,21 @@ def _finish_job_complete_minimal(app_id, app_module):
         les_expected.update({"app_id": app_id, "app_module_name": app_module})
     catalog.log_exec_stats.assert_called_once_with(les_expected)
     mongo.update_job_resources.assert_called_once_with(job_id, resources)
+
+    # Ensure that catalog stats were not logged for a job that was created but failed before running
+    bad_running_timestamps = [-1, 0, None]
+    for timestamp in bad_running_timestamps:
+        log_exec_stats_call_count = catalog.log_exec_stats.call_count
+        job_id2 = "6046b539ce9c58ecf8c3e5f4"
+        subject_job = _finish_job_complete_minimal_get_test_job(
+            job_id2,
+            sched,
+            app_id,
+            gitcommit,
+            user,
+        )
+        subject_job.running = timestamp
+        subject_job.status = Status.created.value
+        sdkmr.get_job_with_permission.side_effect = [subject_job, subject_job]
```

Comment:
Why does there need to be 2 entries here?

Reply:
Per Boris this is a C&P from other test code, so we'll leave as is for now. Should probably figure out what's going on at some point